openmdao/recorders/sqlite_reader.py

```diff
@@ -81,14 +84,28 @@ def __init__(self, filename):
 
         with sqlite3.connect(self.filename) as con:
             cur = con.cursor()
-            cur.execute("SELECT format_version, abs2prom, prom2abs, abs2meta FROM metadata")
+
+            # need to see what columns are in the metadata table before we query it
+            cursor = cur.execute('select * from metadata')
+            names = [description[0] for description in cursor.description]
+            if "var_settings" in names:
+                cur.execute(
+                    "SELECT format_version, abs2prom, prom2abs, abs2meta, var_settings "
+                    "FROM metadata")
+            else:
+                cur.execute(
+                    "SELECT format_version, abs2prom, prom2abs, abs2meta FROM metadata")
             row = cur.fetchone()
             self.format_version = row[0]
             self._abs2prom = None
             self._prom2abs = None
             self._abs2meta = None
+            self._var_settings = None
+
+            if self.format_version == 4:
```

Comment:
you could future proof this with a >=

Reply:
good point. I did that for the >=3 and then forgot to do the same for my new version.

Reply:
should I be waiting for this small change before i merge? @hschilling

Reply:
Sorry, I just forgot to push up. Done.